### PR TITLE
Enable coverage of cram tests

### PR DESCRIPTION
**Pull request by jonludlam**
See https://github.com/ocaml/dune/issues/3884 for some discussion about the problem / workaround

### CODE REVIEW

**Review by Julow — APPROVED** [review]
Works on my machine!

**Comment by jonludlam** [comment]
Didn't work for the coverage reporting done to coveralls via github actions! Fixed that now though.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -60,8 +62,10 @@ jobs:
         if: matrix.send-coverage == true
         run: |
           opam install bisect_ppx
+          mkdir -p $BISECT_FILE
           opam exec -- dune runtest --instrument-with bisect_ppx --force
-          opam exec -- bisect-ppx-report send-to Coveralls
+          opam exec -- bisect-ppx-report send-to Coveralls --coverage-path /tmp/coverage
```
Comment (Julow) [inline .github/workflows/build.yml:67]: Coveralls returns:

```
{"message":"Job #1940.1","url":"https://coveralls.io/jobs/139417004"}
```

Do you have an idea of if we could have this URL easily accessible ?

**Review by Julow — APPROVED** [review]
I had forgotten we ran coveralls for so long as the URL to the result is hard to find.
That works :)

File: .github/workflows/build.yml (hunk 1 above)
Comment (jonludlam) [reply .github/workflows/build.yml:67]: https://coveralls.io/github/ocaml/odoc

**Pull request merged**